ompi/mpi/fortran/mpif-h/alloc_mem_f.c

```diff
@@ -63,14 +65,16 @@ OMPI_GENERATE_F77_BINDINGS (PMPI_ALLOC_MEM_CPTR,
 #pragma weak mpi_alloc_mem_ = ompi_alloc_mem_f
 #pragma weak mpi_alloc_mem__ = ompi_alloc_mem_f
 
-/* Extra pragmas for the _cptr variant from MPI-3.1 */
 #pragma weak MPI_ALLOC_MEM_CPTR = ompi_alloc_mem_f
 #pragma weak mpi_alloc_mem_cptr = ompi_alloc_mem_f
 #pragma weak mpi_alloc_mem_cptr_ = ompi_alloc_mem_f
 #pragma weak mpi_alloc_mem_cptr__ = ompi_alloc_mem_f
 
 #pragma weak MPI_Alloc_mem_f = ompi_alloc_mem_f
 #pragma weak MPI_Alloc_mem_f08 = ompi_alloc_mem_f
+
+#pragma weak MPI_Alloc_mem_cptr_f = ompi_alloc_mem_f
+#pragma weak MPI_Alloc_mem_cptr_f08 = ompi_alloc_mem_f
```

Comment:
Super minor nit: in the above section, it goes:

- pragmas for alloc_mem
- pragmas for alloc_mem_f/_f08
- ...and then all the cptr pragmas

But in this section, it's a different order:
- pragmas for alloc_mem
- pragmas for cptr
- pragmas for alloc_mem_f/f08
- more pragmas for cptr

Can you make the order between these two sections be the same?

(again, I realize this is a super-picky nit -- thank you for your patience!)